database/src/main/java/com/pip/chatbot/repository/CitiesRepository.java

```diff
@@ -14,17 +14,43 @@ public CitiesRepository(DSLContext dsl) {
         this.dsl = dsl;
     }
 
+    public City getCity(String cityName) throws IllegalArgumentException {
+        City city = dsl.selectFrom(Tables.CITY).where(Tables.CITY.CITY_.eq(cityName)).fetchAnyInto(City.class);
+        if (city == null) {
+            throw new IllegalArgumentException();
+        }
+        return city;
+    }
+
     public List<City> getAllCities() {
         return dsl.selectFrom(Tables.CITY).fetchInto(City.class);
     }
 
-    public void createCity(City city) {
+    public City createCity(City city) {
         CityRecord cityRecord = dsl.newRecord(Tables.CITY);
 
         cityRecord.setCity(city.getCity());
         cityRecord.setCountry(city.getCountry());
         cityRecord.setLatitude(city.getLatitude());
         cityRecord.setLongitude(city.getLongitude());
         cityRecord.store();
+        System.out.println(cityRecord.into(City.class));
+        return cityRecord.into(City.class);
+    }
+
+    public void deleteCity(String city) throws IllegalArgumentException {
+        int numberOfRowsAffected = dsl.delete(Tables.CITY).where(Tables.CITY.CITY_.eq(city)).execute();
+        if (numberOfRowsAffected < 1) {
+            throw new IllegalArgumentException();
```

Comment:
Po co nam rzucać tu wyjątek? Jeżeli nic nie było to nie wykona się żadna operacja, a jeżeli było zwrócmy booleana